### ISSUE TITLE
Cannot convert object to Landroid/content/BroadcastReceiver

### ISSUE THREAD

**Issue by rowdyrabouw** [EddyVerbruggen/nativescript-barcodescanner#124]
Follow-up to issue #123 
Android will now return this error with version 2.7.3
```
JS: ERROR Error: Uncaught (in promise): Error: Cannot convert object to Landroid/content/BroadcastReceiver; at index 0
JS: Error: Cannot convert object to Landroid/content/BroadcastReceiver; at index 0
JS:     at onPermissionGranted (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/nativescript-barcodescanner/barcodescanner.js:162:44)
JS:     at file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/nativescript-barcodescanner/barcodescanner.js:205:13
JS:     at new ZoneAwarePromise (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/nativescript-angular/zone-js/dist/zone-nativescript.js:776:29)
JS:     at BarcodeScanner.scan (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/nativescript-barcodescanner/barcodescanner.js:97:16)
JS:     at PensscanComponent.scanQR (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/components/pensscan/pensscan.component.js:62:29)
JS:     at PensscanComponent.ngOnInit (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/components/pensscan/pensscan.component.js:44:14)
JS:     at checkAndUpdateDirectiveInline (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/core/bundles/core.umd.js:10822:19)
JS:     at checkAndUpdateNodeInline (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/core/bundles/core.umd.js:12245:17)
JS:     at checkAndUpdateNode (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/core/bundles/core.umd.js:12184:16)
JS:     at prodCheckAndUpdateNode (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/core/bundles/core.umd.js:12711:5)
JS:     at Object.eval [as updateDirectives] (ng:///AppModule/PensscanComponent_Host.ngfactory.js:19:5)
JS:     at Object.updateDirectives (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/core/bundles/core.umd.js:12625:72)
JS:     at Object.checkAndUpdateView (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/core/bundles/core.umd.js:12151:14)
JS:     at ViewRef_.detectChanges (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/core/bundles/core.umd.js:10225:63)
JS:     at RouterOutlet.activateWith (file:///data/data/nl.nn.mijninkomenvoorlater/files/app/tns_modules/@angular/router/bundles/router.umd.js:5345:42)```

**Comment by EddyVerbruggen** [comment]
Ugh, you're right. Fixing in 2.7.4.

**Comment by rowdyrabouw** [comment]
I can conform it is fixed in 2.7.4 👍
